lib/rules/no-global-jquery.js

```diff
@@ -29,8 +38,8 @@ module.exports = {
       Program() {
         const tracker = new ReferenceTracker(context.getScope());
         const traceMap = {
```

Comment:
We could just move the entire `traceMap` definition to the top of the file so it's all defined together and since it doesn't depend on the rule context. Same for the other rule.